shared/elements/gaia_confirm/style.css

```diff
@@ -85,6 +88,10 @@ p strong {
   font-weight: 300;
 }
 
+.stronger {
+  font-weight: 500;
+}
```

Comment:
the font-weight "300" above is IMO wrong but I don't want to break some uses. That said, I didn't find any place where it's used so maybe I could just change the value above.